hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleAccount.java

```diff
@@ -446,24 +446,29 @@ public void setStakePeriodStart(long stakePeriodStart) {
 	 * Get the num [of shard.realm.num] of node/account this account has staked its Hbar to
 	 * If the returned value is negative it is staked to a node and node num is the absolute value
 	 * If the returned value is positive it is staked to an account.
+	 *
 	 * @return num [of shard.realm.num] of node/account
 	 */
 	public long getStakedNum() {
 		return state().getStakedNum();
 	}
 
 	/**
-	 * Set the node to which this account is staking its Hbar to
+	 * Set the node to which this account is staking its Hbar to. The value of the nodeNum will be negative if its
+	 * staked node id.
+	 *
 	 * @param nodeNum
 	 * 		The node num of the node
 	 */
 	public void setStakedNodeId(long nodeNum) {
```

Comment:
Imo `stakedId` would be a clearer name, with the javadoc updated to note this is an account number when `stakedId > 0` and `-stakedId` is a node id when `stakedId < 0`.

Reply:
Changed `setStakedAccountId` and `setStakedNodeId` to `setStakedId` and changed the parameter name.